fe/fe-core/src/main/java/com/starrocks/catalog/JDBCTable.java

```diff
@@ -180,9 +182,34 @@ private static String buildCatalogDriveName(String uri) {
         // -> jdbc_postgresql_172.26.194.237_5432_db_pg_select
         // requirement: it should be used as local path.
         // and there is no ':' in it to avoid be parsed into non-local filesystem.
-        return uri.replace("//", "")
-                .replace("/", "_")
-                .replace(":", "_");
+
+        String ans = uri.replace("//", "")
```

Comment:
can we remove all non [^0-9a-zA-Z_] chars for safety.

Reply:
I thin it's ok.